recipes/PyAV/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "av" %}
```

Comment:
Sorry!  I didn't know that was something I needed to be looking for :(  @danielballan would you mind clarifying and fixing, please? Thanks!